test_runner/fixtures/zenith_fixtures.py

```diff
@@ -1051,6 +1063,12 @@ def append_pageserver_param_overrides(params_to_update: List[str],
             f'--pageserver-config-override={o.strip()}' for o in env_overrides.split(';')
         ]
 
+    if pageserver_config_override is not None:
```

Comment:
The order of overrides matters, in any params repeat, the last values will be taken.
I propose to move this block upwards, before `env_overrides = os.getenv('ZENITH_PAGESERVER_OVERRIDES')` since IMO the env vars has to to be the ultimate overrides.

Or we could remove the env override functionality entirely?

Reply:
I think that precedence order where explicit CLI params override ENV is more standard.
check for example AWS cli https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html

For the remote_storage use case, it makes sense to override everything using env to enable it for all tests at once.
But if it is specifically overridden in the particular test, we should use this value.